src/Symfony/Component/Cache/Tests/Adapter/AdapterTestCase.php

```diff
@@ -88,7 +88,6 @@ public function testRecursiveGet()
 
         $v = $cache->get('k1', function () use (&$counter, $cache) {
             $v = $cache->get('k2', function () use (&$counter) { return ++$counter; });
-            $v = $cache->get('k2', function () use (&$counter) { return ++$counter; });
```

Comment:
this MUST be kept, it's the core of the test case!

Reply:
Shouldn't the test fail then?

Reply:
It would fail if the implementation were calling the closure a second time.
With your change, if it does, the regression won't be caught.